### PR TITLE
`.gitignore` additions

### PR DESCRIPTION
**Pull request by tmm1**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 **/axolotl.egg-info
 configs
+last_run_prepared
```
Comment (tmm1) [inline .gitignore]: i'll add `qlora-out` too. are there other dirs used by the code?

**Comment by tmm1** [comment]
> i'll add `qlora-out` too. are there other dirs used by the code?

it's kind of all over the place, so maybe we should standardize the examples and use `-model` suffix that can be ignored?

```
README.md:output_dir: ./completed-model
examples/cerebras/qlora.yml:output_dir: ./qlora-out
examples/falcon/config-7b-lora.yml:output_dir: ./falcon-7b
examples/falcon/config-7b-qlora.yml:output_dir: ./qlora-out
examples/falcon/config-7b.yml:output_dir: ./falcon-7b
examples/gptj/qlora.yml:output_dir: ./qlora-out
examples/gptq-lora-7b/config.yml:output_dir: ./llama-7b-lora-int4
examples/jeopardy-bot/config.yml:output_dir: ./jeopardy-bot-7b
examples/llama-2/lora.yml:output_dir: ./lora-out
examples/llama-2/qlora.yml:output_dir: ./qlora-out
examples/mpt-7b/config.yml:output_dir: ./mpt-alpaca-7b
examples/openllama-3b/config.yml:output_dir: ./openllama-out
examples/openllama-3b/lora.yml:output_dir: ./lora-out
examples/openllama-3b/qlora.yml:output_dir: ./qlora-out
examples/pythia-12b/config.yml:output_dir: ./pythia-12b
examples/pythia/lora.yml:output_dir: ./lora-alpaca-pythia
examples/redpajama/config-3b.yml:output_dir: ./redpajama-alpaca-3b
examples/replit-3b/config-lora.yml:output_dir: ./lora-replit
examples/xgen-7b/xgen-7b-8k-qlora.yml:output_dir: ./qlora-out

**Comment by winglian** [comment]
> > i'll add `qlora-out` too. are there other dirs used by the code?
> 
> it's kind of all over the place, so maybe we should standardize the examples and use `-model` suffix that can be ignored?
> 
maybe standardize to `output_dir: ./output-model`?

**Comment by tmm1** [comment]
> maybe standardize to `output_dir: ./output-model`?

can that cause problems if you try to run two different examples in the same checkout? if so we could have `./output-model-llama2` etc?

**Comment by NanoCode012** [comment]
Please add `wandb` folder as well :) ! It becomes large over time.

**Comment by winglian** [comment]
> > maybe standardize to `output_dir: ./output-model`?
> 
> can that cause problems if you try to run two different examples in the same checkout? if so we could have `./output-model-llama2` etc?

or maybe something like `./model-out/<model-name>/`?, then we can simply ignore `model-out`?

**Review by winglian — APPROVED** [review]

**Pull request merged**